code/src/components/reusableStyles.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components/macro';
+
+export const InnerWrapper = styled.main`
+  width: 100vw;
+  
+  @media (min-width: 600px) {
+    width: 80%;
+    max-width: 45rem;
+  }
+`
+export const LocationWrapper = styled.section`
+  padding: 1.5rem;
+  display: flex;
+  flex-direction:column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.6);
+  min-height: 100vh;
```

Comment:
Mobile version looks great, very nice usage of the transparent box around the content when switching to tablet/desktop.